tests/test_ingest.py

```diff
@@ -695,7 +785,9 @@ def test_generate_artifacts(self):
 
         tile = fixture["STAR/en-US"][4]
 
-        data = ingest_links({"STAR/en-US": [tile]}, self.channels[0].id)
+        # making a copy for tiles because distribute will attempt to compress
+        # it in place. That will affect the following tests
+        data = ingest_links({"STAR/en-US": [copy.deepcopy(tile)]}, self.channels[0].id)
```

Comment:
A better way to do this is perhaps to make a deep copy within ingest_links so that the original data is not mutated.
The original version also mutated the input data, and I think that was wrong.